package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kilterset/auth0-actions-testing",
-  "version": "0.3.0",
```

Comment:
Thank you! Can we make this a patch update, `0.3.1` please?

Reply:
https://github.com/kilterset/auth0-actions-testing/pull/32/commits/1cb41f2c8d33e7be864747d24507e41da4d6f4f3